packages/server-express/src/middlewares/session-middleware.ts

```diff
@@ -0,0 +1,92 @@
+import type { TSessionMiddlewareOptions } from '../types';
+import type { Request, Response, NextFunction } from 'express';
+
+import { CLOUD_IDENTIFIERS } from '../constants';
+import expressJwtMiddleware from 'express-jwt';
+import jwksRsa from 'jwks-rsa';
+
+const decodedTokenKey = 'decoded_token';
+
+const mapCloudIdentifierToUrl = (
+  issuer: TSessionMiddlewareOptions['issuer']
+) => {
+  switch (issuer) {
+    case CLOUD_IDENTIFIERS['gcp-au']:
+      return 'https://mc-api.australia-southeast1.gcp.commercetools.com';
```

Comment:
Should we keep these constants in a module in this package of maybe even outside?

Reply:
Maybe later, not sure yet how things will evolve here.